tests/governance_root_ca_certs.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import os
+import sys
+import tempfile
+from cryptography import x509
+import cryptography.hazmat.backends as crypto_backends
+import infra.network
+import infra.path
+import infra.proc
+import infra.net
+import infra.e2e_args
+from infra.proposal import ProposalState
+import suite.test_requirements as reqs
+import ccf.proposal_generator
+
+from loguru import logger as LOG
+
+this_dir = os.path.dirname(__file__)
+
+@reqs.description("Add certificate with quote, query, update")
+def test_cert_store(network, args):
+    # TODO: propose a cert update with valid cert but mismatching mrsigner
```

Comment:
?

Reply:
Yep, it's a TODO :) I wasn't sure how to test this easily. I think I would have to create yet another copy of the gov script with some mismatching MRSIGNER value.

Reply:
Done